core/include/traccc/edm/spacepoint.hpp

```diff
@@ -1,73 +0,0 @@
-/** TRACCC library, part of the ACTS project (R&D line)
```

Comment:
You can use AoS EDM type to keep the original command in adding elements. There is no need to kill them.

Reply:
🤔 No, I think keeping AoS and SoA classes side by side would be very confusing. It would obfuscate even more what is used where.

There should be a way of doing this sort of a thing in [vecmem::edm::host](https://github.com/acts-project/vecmem/blob/main/core/include/vecmem/edm/host.hpp) and [vecmem::edm::device](https://github.com/acts-project/vecmem/blob/main/core/include/vecmem/edm/device.hpp). 🤔